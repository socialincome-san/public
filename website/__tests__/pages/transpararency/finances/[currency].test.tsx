import { render } from '@testing-library/react';
import Finances from '../../../../pages/transparency/finances/[currency]';

describe('Finance page should', () => {
	it('render page unchanged', async () => {
		const { container } = render(<Finances currency={'CHF'} balance={200} />);
		expect(container).toMatchSnapshot();
	});
});
