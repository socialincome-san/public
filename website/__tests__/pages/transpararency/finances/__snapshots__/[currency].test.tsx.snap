// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Finance page should render page unchanged 1`] = `
<div>
  <div
    class="container"
  >
    <main>
      <section>
        <h2>
          Transparency Page
        </h2>
      </section>
      <section>
        <p>
          You are seeing the transparency page for 
          CHF
        </p>
        <p>
          Balance retrieved from firestore: 
          200
        </p>
      </section>
    </main>
  </div>
</div>
`;
